### PR TITLE
added comma for in-text citation

### PR DESCRIPTION
**Pull request by damnation333**
as requested here and pointed out in #1794 https://forums.zotero.org/discussion/55222/citation-style-human-mutation-mistakes/#Comment_241570
Citation disambiguation was tested and works correctly for me.
Journal, book and chapter styles were double checked and seem to work fine.

### CODE REVIEW

**Comment by adam3smith** [comment]
It's possible this is still incorrect wrt to the comma -- did you check? I just meant to point out that the style had been modified since the error report, so you shouldn't expect it to be.

**Comment by damnation333** [comment]
@adam3smith Just checked again. The comma was missing and should defo be there.

File: human-mutation.csl [hunk 1]
```diff
@@ -13,11 +13,11 @@
     <category field="medicine"/>
     <issn>1059-7794</issn>
     <eissn>1098-1004</eissn>
-    <updated>2012-09-27T22:06:38+00:00</updated>
+    <updated>2016-12-27T20:07:23+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <macro name="author-short">
-    <names variable="author">
+    <names variable="author" suffix=",">
```
Comment (rmzelle) [inline human-mutation.csl]: @damnation333, while this works, styles are easier to understand and edit if you put separating punctuation at the highest possible level. In particular, try to write macros in such a way that they don't generate output with dangling separating punctuation (i.e. "Miller et al." is better than "Miller et al.,").

In this case, it's best to change:

```xml
    <layout prefix="(" suffix=")" delimiter="; ">
      <text macro="author-short"/>
      <text macro="issued" prefix=" "/>
    </layout>
```

to

```xml
    <layout prefix="(" suffix=")" delimiter="; ">
      <group delimiter=", ">
        <text macro="author-short"/>
        <text macro="issued"/>
      </group>
    </layout>
```

Comment (damnation333) [inline human-mutation.csl]: great. Thank you for the explanation. I'll keep an eye on these sort of things from now on.

**Comment by adam3smith** [comment]
@damnation333 -- check out the last two changes I made before merging: once you add the delimiter, you'd also remove the affixes (i.e. prefix and suffix) for that author-short and issued macro. Also, make sure indenting stays consistent (for some reason there was an extra level in the sort section of the citation).

Thanks!

**Comment by damnation333** [comment]
@adam3smith Thanks and sorry. So much to learn still.

**Comment by rmzelle** [comment]
Thanks, both.

**Pull request merged**